cdap-unit-test/src/main/java/co/cask/cdap/test/internal/ApplicationManagerFactory.java

```diff
@@ -25,5 +25,5 @@
  */
 public interface ApplicationManagerFactory {
 
-  ApplicationManager create(@Assisted("applicationId") Id.Application applicationId);
+  ApplicationManager create(@Assisted("applicationId") ApplicationId applicationId);
```

Comment:
the package name says ``internal``, but please confirm with @anwar6953 if it is ok to make this change